src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -289,7 +289,8 @@
     <value>Expected a digit ('0'-'9'), but instead reached end of data.</value>
   </data>
   <data name="SpecialNumberValuesNotSupported" xml:space="preserve">
-    <value>.NET number values such as positive and negative infinity cannot be written as valid JSON.</value>
+    <value>.NET number values such as positive and negative infinity cannot be written as valid JSON. To make it work when using 'JsonSerializer', consider specifying 'JsonNumberHandling.AllowNamedFloatingPointLiterals' (see https://docs.microsoft.com/dotnet/api/system.text.json.serialization.jsonnumberhandling).</value>
+  </data>
```

Comment:
This duplicate tag is causing CI failures.

Reply:
Why is this duplicate? I only edited the content of the value-tag?

Reply:
The diff shows this line 293 was added in this change. Maybe something happened in the IDE. It needs to be removed for CI to pass.

Reply:
Oh I have bad eyes 😩. Fixed 👍🏼